code-for-review/create_dictionary.py

```diff
@@ -0,0 +1,26 @@
+def create_dictionary(input1, input2, output):
+    # step 1
+    file1 = open(input1, 'r')
+    file2 = open(input2, 'r')
+
+    # step 2
+    lines1 = file1.readlines()
+    lines2 = file2.readlines()
+
+    file1.close()
+    file2.close()
```

Comment:
Same here, better to catch exception
```suggestion
    # step 1
    try:
        file1 = open(input1, 'r')
        file2 = open(input2, 'r')

        # step 2
        lines1 = file1.readlines()
        lines2 = file2.readlines()

        file1.close()
        file2.close()
    except IOError:
        return "Can't read/write file"
```